ci/plugins/mzcompose/hooks/command

```diff
@@ -51,6 +51,14 @@ if ! mzcompose --mz-quiet list-workflows | grep -q "$service"; then
     mzcompose up -d --scale "$service=0" "$service"
 fi
 
+ci_uncollapsed_heading ":docker: Fetching binaries for coverage"
+mzcompose create
```

Comment:
I think you need to explicitly list the services/containers you want started here. Otherwise a bunch of other containers from the mzcompose.py will be started?

Reply:
`create` does not start the containers but just creates the ones that this test depends on anyway. They will be started later.